src/composer/install.ts

```diff
@@ -2,11 +2,9 @@ import {exec} from '@actions/exec'
 
 export async function install(
   dependencyPreference: string,
-  composerOptions = '',
-  workingDirectory = ''
+  composerOptions = ''
 ): Promise<void> {
   const args: string[] = []
-  const options: string[] = composerOptions.split(' ')
```

Comment:
Related to https://github.com/ramsey/composer-install/pull/107/files#r590790101, you can't use this anymore for parsing options because it breaks when using `--an-option value`.